authentication_sample.js

```diff
@@ -2,15 +2,17 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License. See LICENSE in the project root for
  * license information.
+ * 
+ * This file uses an outdated library.  Please see the readme to find the latest version.
```

Comment:
This statement is not needed anymore since we are now using the latest libraries

```suggestion
```

Reply:
It is updated now. Thank you!